packages/trader/src/Modules/Trading/Components/Elements/purchase-button.jsx

```diff
@@ -61,10 +59,10 @@ const PurchaseButton = ({
                 <Money amount={info.stake} currency={currency} show_currency />
             </Text>
         );
-    } else if (!is_vanilla && !is_turbos) {
+    } else if (!is_vanilla && !is_turbos && !is_accumulator) {
```

Comment:
These conditions are getting harder to read...

Reply:
@akmal-deriv to be honest, it's harder for me to read your suggested version 😅

Reply:
So what version should I let to stay?

Reply:
Up to you) In my opinion, the suggestion is better because if we keep chaining more conditions to it, it would be harder to read as it has negation on every single variable, but by moving negation out we can straight see that all conditions within the block become negative. But it's not crucial to change it though.